Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -185,7 +185,7 @@ HcalDDDRecConstants::getHCID(int subdet, int keta, int iphi, int lay,
     ++phi0;
     unit     = hcons.unitPhi(phibin[eta-1]);
     phi      = hcons.phiNumber(phi0,unit);
-    depth    = hcons.findDepth(subdet,eta,phi,zside,lay);
+    depth    = hcons.findDepth(subdet,eta,phi,zside,lay-1);
```

Comment:
We always use layer # going from 1 to ... But some mapping parameters are stored as 0,.... This was properly taken care of in the Sim step but missed out for Plan 1 operation for TestNumbering option.